app/views/availabilities/index.html.erb

```diff
@@ -0,0 +1,83 @@
+<div class="container-fluid">
+  <h1>Schedule Availability</h1>
+
+  <%= link_to 'Logout', destroy_user_session_path, method: :delete %>
+
+  <%= form_tag("/availabilities", method: "post") do %>
+    <table class="table table-sm">
+      <tr>
+        <th></th>
+        <% start_time = @start_time %>
+        <% end_time = @end_time %>
+        <% while start_time <= end_time do %>
```

Comment:
`start_time.step(end_time, 1.hour)`

Reply:
got this: "can't iterate from ActiveSupport::TimeWithZone"

Reply:
Oh cannot ah. Leave this first I'll figure out some way to fix.

Reply:
Looks like this one can't do anything. :( Just leave it be then.